.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
i dont think we need this file.

Reply:
Do you mean the whole `settings.json`? It was added so I can walk through the `testing` feature of VSCode.